packages/ai/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "private": true,
+  "name": "@webstudio-is/ai",
+  "version": "0.1.0",
+  "description": "Webstudio AI Tools",
+  "author": "Webstudio <github@webstudio.is>",
+  "homepage": "https://webstudio.is",
+  "type": "module",
+  "scripts": {
+    "typecheck": "tsc --noEmit --emitDeclarationOnly false",
+    "checks": "pnpm typecheck && pnpm lint",
+    "dev": "build-package --watch",
+    "build": "pnpm build:prompts && build-package",
```

Comment:
Check `dev` and `build` scripts in project-build. We are gonna get rid of `build-package` cli